shared/src/clients/flamingo-db-client.class/index.ts

```diff
@@ -0,0 +1,35 @@
+import { NodePgDatabase, drizzle } from 'drizzle-orm/node-postgres'
+import { Client } from 'pg'
+import { whatsappSubscribers } from './whatsapp-subscriber'
+import { eq } from 'drizzle-orm'
+
+export default class FlamingoDbClient {
+  private dbUri: string
+  constructor(dbUri: string) {
+    this.dbUri = dbUri
+  }
+
+  private async initializeDb(): Promise<NodePgDatabase> {
+    const client = new Client({
+      connectionString: this.dbUri,
+      ssl: {
+        // required to connect to flamingo db
+        rejectUnauthorized: false,
+      },
+    })
+    await client.connect()
+    return drizzle(client)
+  }
+  public async getAssociatedApiClientId(
+    phoneNumber: string
+  ): Promise<number | null> {
+    const db = await this.initializeDb()
```

Comment:
initializing db here to get around the fact that you cannot have an async constructor.

I'm pretty sure it's not a good idea to initialize and connect to the db on each method call (?) but I'm also not sure what's a better way to deal with this. Stopgap solution, should be fine at a small scale, should remove afterwards?

Reply:
fixed I think